src/atoms/Auth.ts

```
@@ -1,13 +1,79 @@
+    setFeatureFlag({
+      isMapVisible: data.flags?.includes('MAP_VISIBLE') ?? false,
+      isToiletVisible: data.flags?.includes('TOILET_VISIBLE') ?? false,
+    });
+  };
+
+  const setUserInfo = async (user: User) => {
+    _setUserInfoAtom(user);
+    _syncUserInfo(user);
+  };
+
+  const syncUserInfo = async () => { userInfo && await _syncUserInfo(userInfo) }
```

ESLint

[error] Replace `·userInfo·&&·await·_syncUserInfo(userInfo)·}` with `⏎····userInfo·&&·(await·_syncUserInfo(userInfo));⏎··};`

(prettier/prettier)